fdbrpc/genericactors.actor.h

```diff
@@ -128,8 +126,29 @@ ACTOR template <class T> Future<Void> broadcast( Future<T> input, std::vector<Re
 	return Void();
 }
 
+ACTOR template <class T> Future<Void> incrementalBroadcast(Future<T> input, std::vector<Promise<T>> output, int batchSize) {
+	state T value = wait(input);
+	state int i = 0;
+	for (; i<output.size(); i++) {
+		output[i].send(value);
+		if((i+1)%batchSize==0) {
+			wait(delay(0));
+		}
+	}
+	return Void();
+}
 
-
+ACTOR template <class T> Future<Void> incrementalBroadcast( Future<T> input, std::vector<ReplyPromise<T>> output, int batchSize) {
+	state T value = wait( input );
+	state int i = 0;
+	for(; i<output.size(); i++) {
+		output[i].send(value);
+		if((i+1)%batchSize==0) {
+			wait(delay(0));
+		}
+	}
+	return Void();
+}
```

Comment:
Is GitHub being weird or is this actually repeated twice?

Reply:
Different type of `output`

Reply:
But it does feel like `ReplyPromise` probably should be a sub class of `Promise`, which is not the case. That's a bit unfortunate.

Reply:
Oh, good reading.